### PR TITLE
Add default for Automatic Allocation in CreateNode

### PR DESCRIPTION
**Pull request by RMartinOscar**
Add ->default matching env PANEL_CLIENT_ALLOCATIONS_ENABLED to avoid 500 when creating a node and Automatic Allocation is not set

### CODE REVIEW

File: app/Filament/Resources/NodeResource/Pages/CreateNode.php [hunk 1]
```diff
@@ -212,6 +212,7 @@ public function form(Forms\Form $form): Forms\Form
                                     false => 'success',
                                 ]),
                             Forms\Components\ToggleButtons::make('public')
+                                ->default(config('panel.client_features.allocations.enabled', false))
```
Comment (Boy132) [inline app/Filament/Resources/NodeResource/Pages/CreateNode.php:215]:
```suggestion
                                ->default(true)
```

This has nothing to do with "automatic allocations". `public` only controls if a node can be used for deployment.
"automatic allocations" is the feature that let users create a random allocation in the client area. (yes, the naming is a bit confusing)

**Review by Boy132 — CHANGES_REQUESTED** [review]

File: app/Filament/Resources/NodeResource/Pages/CreateNode.php (hunk 1 above)
Comment (Boy132) [reply app/Filament/Resources/NodeResource/Pages/CreateNode.php:215]: If you want you can change the label to something like `Useable for deployment`  to make it clearer and add a helper text that explains the field.

**Comment by RMartinOscar** [comment]
Mybad i think it's ready to merge now

**Review by Boy132 — APPROVED** [review]
LGTM

**Pull request merged**